### ISSUE TITLE
TypeError: hoistStatics is not a function

### ISSUE THREAD

**Issue by techwes** [reactivestack/cookies#293]
This morning on of my production servers spun up and pulled down the latest version of `react-cookie` and started giving 500s due to the following error:
```
/var/app/current/node_modules/react-cookie/cjs/withCookies.js:146
return hoistStatics(ForwardedComponent, WrappedComponent);
^
TypeError: hoistStatics is not a function
at withCookies (/var/app/current/node_modules/react-cookie/cjs/withCookies.js:146:10)
at /var/app/current/dist/common/cookies.js:86:92
```
We were able to successfully resolve the issue by pinning the version of `react-cookie` we were depending on to the previous version. We are server-side rendered on Node 12 and use Webpack to bundle our client side files. Let me know what other information you might need to debug the issue.

**Comment by eXon** [comment]
The issue was most likely introduced in v4.1.0. You can downgrade until it is fixed. I should be able to investigate and have a fix quickly.

**Comment by techwes** [comment]
Thank you. I agree it was certainly introduced in 4.1.0 and downgrading did indeed fix the issue

**Comment by eXon** [comment]
This is fixed in v4.1.1.
